plugins/shuffle/qt.py

```diff
@@ -1301,13 +1296,8 @@ def setup(self):
         l.addItem(QSpacerItem(1, 1, QSizePolicy.MinimumExpanding, QSizePolicy.Fixed), 1, 5)
 
 
-        icon = QLabel(self)
-        if not SendTabExtra.pixmap_cached:
-            # cache it and keep it around, since scaling this pixmap wastes CPU cycles each time
-            size = QSize(125, int(125/1.4419))
-            # NB: doing it this way, with a QIcon, will take into account devicePixelRatio and end up possibly producing a very hi res image from the SVG, larger than size
-            SendTabExtra.pixmap_cached = QIcon(":icons/CashShuffleLogos/logo-vertical.svg").pixmap(size)
-        icon.setPixmap(SendTabExtra.pixmap_cached)
+        icon = FixedAspectRatioSvgWidget(75, ":icons/CashShuffleLogos/logo-vertical.svg")
```

Comment:
This was supposed to be 125. I'll fix it now. :)

Reply:
Ooops, I blame it on copy & paste